plugins/module.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+module.exports = function (opts) {
+	const modules = opts && opts.modules !== null ? opts.modules : true;
```

Comment:
Can we treat this the same as https://github.com/babel/babel-preset-env#modules? E.g. the only allowed values are `'commonjs'` or `false`.